torchci/pages/kpis.tsx

```diff
@@ -25,7 +26,83 @@ export default function Kpis() {
     
     return (
         <Grid container spacing={2}>
-            <Grid item xs={true} height={ROW_HEIGHT}>
+            <Grid item xs={6} height={ROW_HEIGHT}>
+                <TimeSeriesPanel
+                title={"Avg Time To Signal - E2E (Weekly)"}
+                queryName={"time_to_signal"}
+                queryCollection={"pytorch_dev_infra_kpis"}
+                queryParams={[
+                    {
+                        name: "buildOrAll",
+                        type: "string",
+                        value: "all",
+                    },
+                    ...timeParams,
+                ]}
+                granularity={"week"}
```

Comment:
Since this is reused a bunch, you can do something like 
```
const timeSeriesProps = {
granularity: "week",
timeFieldName:"week_bucket",
...
}

<TimeSeriesPanel
   ...timeSeriesProps
/>
```

Reply:
it's only the granularity that keeps getting reused. The time filed name changes based on the query.  Extracting just the granularity feels overly DRY for now (that may change later though)